index.js

```diff
@@ -104,7 +106,12 @@ function finalhandler(req, res, options) {
       return req.socket.destroy()
     }
 
-    send(req, res, status, msg)
+    if(typeof render === 'function'){
+      render(msg, req, res, function(rendered){
```

Comment:
The signature `function(rendered)` is not acceptable here; it means an error could never possibly occurs within rendering and does not property fit in with error-first callback pattern. Please have it accept `err` as a first argument and then act on that error if one is provided.